### PR TITLE
✨ Mention renovatebot's settings

### PR DESCRIPTION
**Pull request by laurentsimon**
Explicitly say that renovatebot can help migrate from version pinning to hash pinning via a setting.
See https://github.com/dependabot/dependabot-core/issues/3699#issuecomment-1025715911

### CODE REVIEW

**Review by naveensrinivasan — APPROVED** [review]
Thank you!

File: docs/checks/internal/checks.yaml [hunk 1]
```diff
@@ -78,6 +78,10 @@ checks:
         be enabled for forks where security updates have ever been turned on so projects
         maintaining stable forks should evaluate whether this behavior is satisfactory
         before turning it on.
+      - >-
+        Unlike dependabot, renovatebot has support to migrate dockerfiles' dependencies from version pinning to hash pinning
```
Comment (azeemshaikh38) [inline docs/checks/internal/checks.yaml:82]: Nit: might come across as if we are putting down dependabot. Consider changing to:

```
Renovatebot supports migrating Dockerfile dependencies from ...
```

**Review by azeemshaikh38 — APPROVED** [review]

File: docs/checks/internal/checks.yaml (hunk 1 above)
Comment (azeemshaikh38) [reply docs/checks/internal/checks.yaml:82]: I thought `conversation resolution before merging` should stop this PR from being merged, lol.

Comment (laurentsimon) [reply docs/checks/internal/checks.yaml:82]: it should have!

**Pull request merged**